include/qdmi/constants.h

```diff
@@ -513,6 +513,13 @@ enum QDMI_OPERATION_PROPERTY_T {
   QDMI_OPERATION_PROPERTY_DURATION = 3,
   /// `double` The fidelity of an operation.
   QDMI_OPERATION_PROPERTY_FIDELITY = 4,
+  /**
+   * @brief `bool` An operation is global if it acts on all sites
+   * simultaneously. the operation is applicable to.
```

Comment:
```suggestion
   * @brief `bool` An operation is global if it acts on all sites
   * the operation is applicable to simultaneously.
```
Something was not quite right here.